packages/inventory/src/components/table/EntityTable.js

```diff
@@ -62,6 +67,16 @@ const EntityTable = ({
         onSort?.({ index, key, direction });
     };
 
+    const disabledColumns = Array.isArray(disableDefaultColumns) ? disableDefaultColumns : [];
+    const defaultColumnsFiltered = defaultColumns.filter(({ key }) =>
+        (key === 'tags' && showTags) || (key !== 'tags' && !disabledColumns.includes(key))
+    );
+
+    const columns = mergeArraysByKey([
+        typeof disableDefaultColumns === 'boolean' && disableDefaultColumns ? [] : defaultColumnsFiltered,
+        columnsProp || columnsRedux || []
+    ], 'key');
```

Comment:
This is quite a lot of heavy computing, might be reasonable to move all of this `useEffect` where you could use the check from `useSelector` of `columnsRedux` and store the value in `ref`. We don't want to trigger yet another re-render, but the calculation could be usefull to moize this way. Or we can use `useMemo`. But the memoize function would be a bit complex.

Either way it should be reasonable to somehow limit the amount of calculations for this piece of code as it uses `filter` and `mergeArraysByKey` at each re-render.

Reply:
done!